app/models/user.rb

```diff
@@ -202,6 +202,25 @@ def soft_deleted?
     deleted_at.present?
   end
 
+  def associated_schools
+    scope = School.distinct
```

Comment:
This this would be better
```suggestion
    scope = School.gias_status_open.distinct
```